lib/awful/mouse/init.lua

```diff
@@ -298,12 +298,42 @@ function mouse.object.set_current_client() end
 function mouse.object.get_current_wibox()
     local obj = capi.mouse.object_under_pointer()
     if type(obj) == "drawin" then
-        return obj.wibox()
+        return obj.wibox
```

Comment:
Another bad cherry pick? (This should actually be in the "fix `current_wibox`"-commit and I didn't even notice this error...)